composer.json

```
@@ -3,7 +3,7 @@
     "description": "Amwal payment plugin for Magento 2 enables one click checkout for instantaneous purchases from product, mini-cart and cart pages.",
     "homepage": "https://amwal.tech",
     "type": "magento2-module",
-    "version": "1.0.32",
+    "version": "1.0.33",
```

Let's maybe use a specific version to avoid conflicts in the future
```suggestion
    "version": "1.0.32-php73",
```